### DISCUSSION TITLE
[Icon Requuest] Coins, People, Banks, Slashed Circles

### DISCUSSION THREAD

**Discussion by zackkh** [tailwindlabs/heroicons#908] [Ideas]
Is it possible to get these icons

- Coins
- People (Male and Female)
- Banks
- Slashed Circle (to represent ban action)
- add `archive-box-arrow-up` and `?floppy-disk`

Thank You.

**Comment by jbasko** [comment]
I'm also a bit surprised to discover that there are no male/female or man/woman icons in this. Is this part of some progressive ideology?
